src/core/renderers/webgl/WebGLRenderer.js

```diff
@@ -56,14 +56,16 @@ export default class WebGLRenderer extends SystemRenderer
      *  with older / less advanced devices. If you experiance unexplained flickering try setting this to true.
      * @param {string} [options.powerPreference] - Parameter passed to webgl context, set to "high-performance"
      *  for devices with dual graphics card
+     * @param {Boolean} [options.forceNative=false] - Whether use the native methods or the WebGL extension.
```

Comment:
Documentation should also be duplicated in Application and autoDetectRenderer

Reply:
and it would be boolean, not Boolean

Reply:
done.